server/app/models/grid_service.rb

```diff
@@ -43,6 +43,8 @@ class GridService
   belongs_to :grid
   belongs_to :image
   belongs_to :stack
+  belongs_to :grid_domain_authorization
```

Comment:
This only allows one `GridDomainAuthorization` per (linked LB) service? If you want to have this field, then it needs to be the other way around: `GridDomainAuthorization belongs_to :grid_service`, and `GridService has_many :grid_domain_authorizations`. It should probably also be called something like `:linked_service` to make it clearer.

It doesn't look like this relation is actually used for anything, though.